website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+
+#### Context-registered broadcasts
+
+When using SDK >= 34, we've migrated [Broadcast Receivers](https://developer.android.com/guide/components/activities/process-lifecycle) to support context-registered broadcasts — part of the app lifecycle model — in Android 14. This means React Native apps will behave predictably on Android 14 devices.
+
+#### Target SDK version
+
+In 0.73, React Native developers can now target the latest Android SDK version, [API Level 34](https://developer.android.com/guide/topics/manifest/uses-sdk-element?hl=en#ApiLevels) (*Upside Down Cake*).
+
+### Deprecated Debugging Features
+
+#### Flipper ↔ React Native integration
+
+We are gradually moving away from Flipper as the default tool for debugging React Native apps. In 0.73, this begins by deprecating the native Flipper integration included with React Native (bootstrap code which wires up Flipper's core plugins). We will be removing this integration and dependency in the next release — meaning parts of Flipper such as the Network plugin will cease to work.
+
+**Unchanged**: Flipper as a standalone product for native app debugging will continue to exist. Even after its removal from new React Native projects in future, developers will be able to [manually add Flipper to their app](https://fbflipper.com/docs/getting-started/android-native/) if they wish.
+
+For more information on why we are moving away from Flipper, [view the RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0641-decoupling-flipper-from-react-native-core.md).
+
+#### Remote JavaScript Debugging
+
+Remote JavaScript Debugging is a legacy debugging mode that connects an external web browser (Chrome) to your app and runs your JavaScript code inside a web page, i.e. `http://localhost:8081/debugger-ui`. This model could lead to inconsistent app behaviour while debugging, and is incompatible with native modules under the New Architecture.
+
+In 0.73, Remote JavaScript Debugging is deprecated and has been removed from the Dev Menu. Enabling the remote debugger must now be done manually via the `NativeDevSettings` API. Doing this is covered in the [Other Debugging Methods docs](/docs/next/other-debugging-methods#remote-js-debugging).
+
+:::info
+Remote JavaScript Debugging was previously the default debugging experience for apps using JavaScriptCore (JSC). We recommend [Safari Developer Tools (direct JSC debugging)](/docs/next/other-debugging-methods#safari-developer-tools-direct-jsc-debugging) instead, for iOS apps.
+
+We recommend using [Hermes](/docs/hermes) for a consistent debugging experience on all platforms.
+:::
+
+### New Architecture Updates
+
+We continue active development on the New Architecture and are fully invested in bringing it to the OSS community. You can read the latest updates in the [New Architecture working group](https://github.com/reactwg/react-native-new-architecture/discussions).
```

Comment:
I know we don't want to talk much about this in the blog post, but I think it would be helpful if we could say more. I think people have questions, and looking at the link doesn't really talk to that progress or answer any questions they might have

Reply:
@huntie agree. We need 2x paragraphs here:
1. One on Bridgeless/SVC being available for experimentation
2. One on Native Module interop
And they should link to the post we'll be posting in the New Architecture WG

Reply:
I've done a first pass at this, with some input from @dmytrorykun.